changelog/4598.docfix.rst

```diff
@@ -0,0 +1 @@
+Modified :ref:`sphx_glr_generated_gallery_plotting_magnetogram_active_regions_example.py` to use HMI file from sample data instead of downloading it with Fido.
```

Comment:
```suggestion
Modified :ref:`sphx_glr_generated_gallery_plotting_magnetogram_active_regions.py` to use HMI file from sample data instead of downloading it with Fido.
```